freeCodeCamp/freeCodeCamp
# feat(client, tools): make ui component respect theme

Checklist:





To test this PR

- Link the ui-component to the client 
- build the ui-component in `./tools/ui-component`
- import a component that need night and light themes from the ui-component, most of the buttons or dropdown components
- check the page in light and dark theme
